### PR TITLE
V1

### PR DESCRIPTION
**Pull request by fapfap69**
FIrst version

### CODE REVIEW

**Comment by fapfap69** [comment]
Please could you review the PR, I'm not able to calculate pedestals . Thank you. @chiarazampolli @martenole @shahor02 @davidrohr

**Comment by chiarazampolli** [comment]
Hi @fapfap69 , 

I cannot spot any issue with your PR, but maybe it is better if also @davidrohr takes a look.

Chiara

File: DATA/production/calib/hmp-pedestals-processing.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#
+#
+#   Auth. A.Franco  - INFN  Sez.BARI - ITALY
+# ------------------------------------------------------------------------
+
+source common/setenv.sh
+# env  > /home/fap/dump_env.txt
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Set general arguments
```
Comment (davidrohr) [inline DATA/production/calib/hmp-pedestals-processing.sh:21]: I would recommend to source `getCommonArgs.sh` instead of copy&pasting this part

Comment (fapfap69) [reply DATA/production/calib/hmp-pedestals-processing.sh:20]: Done ! Thank you

Comment (davidrohr) [reply DATA/production/calib/hmp-pedestals-processing.sh:20]: Did you test this? getCommonArgs is in the common folder as is setenv.sh, so I assume you have to do `source common/getCommonArgs.sh`.
Did you test this workflow at P2? You can check out a local clone with this PR to your home folder on the EPN, and then use the workflow configuration method = repository path pointing to your DATA path to test it, before merging to the official repository.

Comment (fapfap69) [reply DATA/production/calib/hmp-pedestals-processing.sh:20]: yes, i did all the test in my DATA path before the PR ...

**Comment by fapfap69** [comment]
> Il giorno 22 lug 2022, alle ore 10:16, David Rohr ***@***.***> ha scritto:
> 
> 
> @davidrohr commented on this pull request.
> 
> In DATA/production/calib/hmp-pedestals-processing.sh <https://github.com/AliceO2Group/O2DPG/pull/523#discussion_r927408272>:
> 
> > +#
> +#  Extra Env Variables:
> +#     HMP_SIGMACUT : The value of sigams for the threshold cut [=4]
> +#     HMP_CCDB_REC : True if we want the recording into ALICE CCDB
> +#     HMP_PED_TAG :  A string that tags the pedestals [=Latest]
> +#
> +#
> +#
> +#   Auth. A.Franco  - INFN  Sez.BARI - ITALY
> +# ------------------------------------------------------------------------
> +
> +source common/setenv.sh
> +# env  > /home/fap/dump_env.txt
> +
> +# ---------------------------------------------------------------------------------------------------------------------
> +# Set general arguments
> getCommonArgs is in the common folder as is setenv.sh, so I assume you have to do source common/getCommonArgs.sh.
> Did you test this workflow at P2? You can check out a local clone with this PR to your home folder on the EPN, and then use the workflow configuration method = repository path pointing to your DATA path to test it, before merging to the official repository.
> 

Yes I did some pedestals calculation at P2 in my private folder, it works, clearly I need to tune better the different configurations and the write into the CCDB...

**Pull request merged**